tool/site/client/src/pages/claimPage.tsx

```diff
@@ -83,11 +93,96 @@ async function retrievePrescriptionDetails(baseUrl: string, prescriptionId: stri
     throw new Error("Dispense notification not found. Has this prescription been dispensed?")
   }
 
-  return {
+  const prescriptionDetails = {
     patient: getPatientResources(prescriptionOrder)[0],
     medicationRequests: getMedicationRequestResources(prescriptionOrder),
     medicationDispenses: dispenseNotifications.flatMap(getMedicationDispenseResources)
   }
+
+  if (amend) {
+    const claimResponse = await axiosInstance.get<fhir.Claim>(`${baseUrl}claim/${prescriptionId}`)
+    const claim = getResponseDataIfValid(claimResponse, isClaim)
+    return {
+      ...prescriptionDetails,
+      claim
+    }
```

Comment:
rather than a seperate return we could just do `prescriptionDetails.claim = claim`, types might kick off but if we added a type on line 96 that'd go away

Reply:
good shout, will add